### ISSUE TITLE
PDF rendering

### ISSUE THREAD

**Issue by EvgeniiaVak** [datopian/datapackage-views-js#47]
PDF rendering can be done by browser - we would just need to embed the object tag with pdf. 
e.g. with an oddk file:
```html
<object data="https://cc-p-minio.ckan.io/ckan/opendatadenmark-prod/resources/3f2d5f2b-8659-4995-926a-8fe77798daff/bilag-1-trafiktaellinger-kobenhavnskommune.pdf?AWSAccessKeyId=2effdd1004072cb9&Expires=1596084974&Signature=2Q95679q22UfImIKDbiGEU63Enw%3D" width="400" height="500"></object>
```

Why are we doing PDF rendering in `datapackage-views-js`? 

One hypothesis is to support pagination.

**Comment by EvgeniiaVak** [comment]
refs #46

**Comment by EvgeniiaVak** [comment]
cc @anuveyatsu

**Comment by anuveyatsu** [comment]
@EvgeniiaVak I agree that we can/should simplify PDF rendering logic. Having pagination might be important feature for some users. I personally would just have a link to a PDF file and I could preview it using my own tool.

Note `datapackage-views-js` running in browser as it's a React app.
